src/logic/function/basic.lean

```diff
@@ -86,7 +86,7 @@ h ▸ hf.ne_iff
 
 /-- If the co-domain `β` of an injective function `f : α → β` has decidable equality, then
 the domain `α` also has decidable equality. -/
-def injective.decidable_eq [decidable_eq β] (I : injective f) : decidable_eq α :=
+protected def injective.decidable_eq [decidable_eq β] (I : injective f) : decidable_eq α :=
```

Comment:
Was there a reason for making this change? It's one more thing to backport, and also slows the builds down a lot

Reply:
This was merged to master separately in #15759. The main reason is that I didn't want to have to write `_root_.decidable_eq` in the arguments of `comap_lp_single` (which is in `namespace function.injective`). It's not `protected` in Mathlib4, but I'd say the right thing to do is just to leave things as is for now, and we'll have to decide what to do (either make it protected in Mathlib4 or find a way around it) when porting `lp_space.lean`.

Reply:
How do you know it slows the build time ?

Reply:
Oh, I hadn't realized it was already merged. In that case, I have no complaint at all :)